framework/test/jest/unit/specs/modules/chain/dpos/apply.spec.js

```diff
@@ -179,6 +214,18 @@ describe('dpos.apply()', () => {
 				)
 				.mockResolvedValue(delegatesWhoForged);
 
+			when(stubs.storage.entities.Account.get)
+				.calledWith(
+					{
+						isDelegate: true,
+					},
+					{
+						limit: constants.ACTIVE_DELEGATES,
+						sort: ['voteWeight:desc', 'publicKey:asc'],
+					},
+				)
+				.mockResolvedValue(delegateAccounts);
```

Comment:
I'm not sure if these accounts are sorted by voteweight and publickey can you make sure that array is sorted.